server/src/main/java/org/opensearch/http/DefaultRestChannel.java

```diff
@@ -69,13 +71,16 @@ public class DefaultRestChannel extends AbstractRestChannel implements RestChann
     static final String CONTENT_TYPE = "content-type";
     static final String CONTENT_LENGTH = "content-length";
     static final String SET_COOKIE = "set-cookie";
+    static final String SERVER_VERSION = "server-version";
+    static final String OPEN_SEARCH_NAME = "OpenSearch";
 
     private final HttpRequest httpRequest;
     private final BigArrays bigArrays;
     private final HttpHandlingSettings settings;
     private final ThreadContext threadContext;
     private final HttpChannel httpChannel;
     private final CorsHandler corsHandler;
+    private final Map<String, List<String>> serverVersion;
```

Comment:
This is not needed

Reply:
It is needed, as addCustomHeaders needs a Map field to take as parameter to add the header. Dont believe theres a way to do this inline.

Reply:
`Collection.singletonMap`?

Reply:
Moved it so its defined just above the put statement agree doesnt need to be global

Reply:
The more you know only recently got back into using Java so appreciate the pointers! :)